packages/net/tcp_connection.pony

```diff
@@ -748,7 +748,7 @@ actor TCPConnection
   fun ref _apply_backpressure() =>
     ifdef not windows then
       _writeable = false
-      ifdef linux then
+      ifdef linux or freebsd or osx then
```

Comment:
as this is in a "not windows" block, we no longer need this ifdef. it was there to keep it from running on freebsd or osx.